ci/pbs.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -x
+
+function jobqueue_before_install {
+    docker version
+    docker-compose version
+
+    # start pbs cluster
+    cd ./ci/pbs
+    docker-compose up -d
+    while [ `docker exec -it -u pbsuser pbs_master pbsnodes -a | grep "Mom = pbs_slave" | wc -l` -ne 2 ]
```

Comment:
Actually, locally I just use the commands
```bash
cd ci/pbs
docker-compose up -d
```
I can then see when the cluster is started. I feel there is no need of a script to do this.

I would prefer to avoid having too many scripts, I feel that it is easier to understand this way. When somebody new looks at the Travis build config file, he does'nt have to go through 2 other ones just for the before_install phase.

If we want to make things easier to test locally, may be we should add a script which call in order all the jobqueue_ * from pbs.sh, or better, any jobquepue.sh file.

Reply:
> I would prefer to avoid having too many scripts, I feel that it is easier to understand this way. When somebody new looks at the Travis build config file, he does'nt have to go through 2 other ones just for the before_install phase.

I totally get your point that using scripts adds another level of indirection. I think we are coming at it from a different point of view: I have looked at quite a few different .travis.yml files and to me the dask-jobqueue is nicely layed out and straightforward to follow. What I am a newbie about though is the docker side and I feel this may be the case for other potential contributers/users of dask-jobqueue. Being able to just run a bash script and not have to know too much about docker is a great advantage.

On top of the `docker-compose up -d, my understanding ` the while loop is to check that the cluster is correctly set-up. IIRC when I was working on the SGE CI, there could be a delay of more than 1 minute before you exit the while loop and it's nice to have something at the end confirming that everything went fine.

Also I think having consistency between the different schedulers CI setup is very important.

Reply:
Okay, you got me with the last consistency argument.